packages/clerk-js/src/ui/elements/UserPreview.tsx

```diff
@@ -68,16 +63,15 @@ export const UserPreview = (props: UserPreviewProps) => {
         >
           {name || identifier} {badge}
         </Text>
-        {name && identifier && (
+        {(subtitle || (name && identifier)) && (
```

Comment:
❓ Do we want the subtitle to show even if there is no name?

Reply:
That's a valid point George. Leaving this as is for now because I have another open branch with changes on this, will tackle this concern there directly. Thanks :)